Toolbar customization/InsertToolbarItems/README.md

```diff
@@ -0,0 +1,245 @@
+# How to include Open and Save options in the built-in toolbar?
+
+This project demonstrate how to insert the save and open options in the toolbar in [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) and how to create and it also showcases how to create and handle a custom DocumentSaveInitiated event.
+
+## Prerequisites
+
+1. A .NET MAUI project set up.
+2. The [Syncfusion.Maui.PdfViewer](https://www.nuget.org/packages/Syncfusion.Maui.PdfViewer) package is installed.
+
+## To Include open and save option in built-in toolbar
```

Comment:
<img width="864" height="106" alt="image" src="https://github.com/user-attachments/assets/ead18bc7-7fb5-4ee1-bf79-81b986fed9cd" />

Before defining a clicked event, its event handler should not be exposed.

I have previously asked you to implement logic for Open and Save document. You may create as separate method under the topic. Later, call the method in the event handler later.